content/syllabuses/k8s.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Kubernetes (A.K.A K8s)
+---
+
+- {{< contentlink path="k8s/introduction" >}}
+- {{< contentlink path="k8s/manual-app-deployment/project-overview" >}}
+- {{< contentlink path="k8s/manual-app-deployment/ec2-setup" >}}
+- {{< contentlink path="k8s/manual-app-deployment/certbot" >}}
+- {{< contentlink path="k8s/manual-app-deployment/nginx-installation" >}}
+- {{< contentlink path="k8s/manual-app-deployment/postgresql-installation" >}}
+- {{< contentlink path="k8s/manual-app-deployment/github-setup" >}}
+- {{< contentlink path="k8s/manual-app-deployment/nginx-tls" >}}
+- {{< contentlink path="k8s/manual-app-deployment/python-app-setup" >}}
+- {{< contentlink path="k8s/manual-app-deployment/nginx-reverse-proxy" >}}
+- {{< contentlink path="k8s/containers-with-docker/containers-overview" >}}
+- {{< contentlink path="k8s/containers-with-docker/certbot" >}}
+- {{< contentlink path="k8s/containers-with-docker/dockerfile" >}}
+- {{< contentlink path="k8s/containers-with-docker/docker-compose" >}}
+- {{< contentlink path="k8s/containers-with-docker/nginx" >}}
+- {{< contentlink path="k8s/containers-with-docker/python-app" >}}
+- {{< contentlink path="k8s/containers-with-docker/running" >}}
+- {{< contentlink path="k8s/kubernetes/deploy-app-with-manifest-file" >}}
```

Comment:
The correct ordering here is:

```
- {{< contentlink path="k8s/kubernetes/k3s-quickstart" >}}
- {{< contentlink path="k8s/kubernetes/expanding-the-cluster" >}}
- {{< contentlink path="k8s/kubernetes/deploy-app-with-manifest-file" >}}
```

Also, how will the ordering between pages under each module work? Through prerequisites? Or should we make them explicit here?